tests/features/solution/add_solution.feature

```diff
@@ -37,6 +37,8 @@ Feature: "Add solution" visibility options.
       | Description       | This is a test text              |
       | Documentation     | text.pdf                         |
     And I press "Save"
-    Then the "Collection solution test 2" collection has a solution named "Collection solution add solution"
+    # The name of the solution should exist in the block of the relative content in a collection.
+    When I go to the homepage of the "Collection solution test 2" collection
+    Then I should see the link "Collection solution add solution"
```

Comment:
What is shown on this page? I guess we don't have the specs / wireframes of this yet. Maybe just click it and assert that the title is showing correctly, then at least we can already catch any fatal errors that occur on that page.